translations/monero-core_fr.ts

```diff
@@ -792,628 +762,807 @@
     </message>
     <message>
-        <location filename="../pages/Receive.qml" line="270"/>
-        <source>Clear</source>
-        <translation>Effacer</translation>
+        <location filename="../pages/Receive.qml" line="389"/>
+        <source>(Untitled)</source>
+        <translation>(Sans titre)</translation>
     </message>
     <message>
-        <location filename="../pages/Receive.qml" line="279"/>
-        <source>Integrated address</source>
-        <translation>Adresse intégrée</translation>
+        <location filename="../pages/Receive.qml" line="409"/>
+        <source>Advanced options</source>
+        <translation>Options avancés</translation>
     </message>
     <message>
-        <location filename="../pages/Receive.qml" line="300"/>
-        <source>Integrated address copied to clipboard</source>
-        <translation>Adresse intégrée copiée dans le presse-papier</translation>
+        <location filename="../pages/Receive.qml" line="429"/>
+        <location filename="../pages/Receive.qml" line="434"/>
+        <source>QR Code</source>
+        <translation>Code QR</translation>
     </message>
     <message>
-        <location filename="../pages/Receive.qml" line="320"/>
+        <location filename="../pages/Receive.qml" line="525"/>
+        <source>&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;This is a simple sales tracker:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Let your customer scan that QR code to make a payment (if that customer has software which supports QR code scanning).&lt;/p&gt;&lt;p&gt;This page will automatically scan the blockchain and the tx pool for incoming transactions using this QR code. If you input an amount, it will also check that incoming transactions total up to that amount.&lt;/p&gt;It&apos;s up to you whether to accept unconfirmed transactions or not. It is likely they&apos;ll be confirmed in short order, but there is still a possibility they might not, so for larger values you may want to wait for one or more confirmation(s).&lt;/p&gt;</source>
+        <translation>&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;Voici un suivi de commande simplifi&eacute; :&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Permet &agrave; un client de scanner le QR code poue effectuer un paiement (si ce client dispose d'une application supportant le scan de QR code).&lt;/p&gt;&lt;p&gt;Cette page va automatiquement scanner la cha&icirc;ne de blocs et le pool de transaction pour trouver la transaction entrante en utilisant ce QR code. Si vous saisissez un montant, elle va &eacute;galement vérifier que la somme des transactions entrantes correspond à ce montant.&lt;/p&gt;A vous de pr&eacute;ciser si vous acceptez les transactions non-confirm&eacute;es ou non. Il est probable qu&apos;elles soiet confirmer rapidement, mais il y a toujours un risque qu&apos;elles ne le soient pas. Vous pourriez donc souhaiter attendre une ou plusieurs confirmations pour les montants les plus &eacute;lev&eacute;s.&lt;/p&gt;</translation>
```

Comment:
Please remove entity `&eacute;`

Reply:
why? &eacute; is for displaying "simplifié" with a "é"
However please correct "d'une": s/d\'une/d&apos;une/g

Reply:
Or is it only an escape problem for quotes, and not a charset issue? If so, there are also è ê î and à which may be simplified.

Reply:
it's a charset problem. Only a small set of entity are supported, but they are automatically generetated and we really don't need to use them. Let's use the simplified version, it will be correctly picked. see [my comment](https://github.com/monero-project/monero-gui/pull/1390#pullrequestreview-119379976)

Reply:
ok. Sorry for the mess i created. It was a pain writing with those btw.

Reply:
I imagine, it was also a pain to spot them all. Please, next time test your translations before submitting them. You can do so using the `lrelease` command or a graphic interface like Linguist.